lib/open_food_network/order_cycle_management_report.rb

```diff
@@ -77,7 +77,7 @@ def payment_method_row(order)
        ba.phone,
        order.shipping_method.andand.name,
        order.payments.first.andand.payment_method.andand.name,
-       order.payment_total,
+       order.payments.first.amount,
```

Comment:
how would that handle the case of multiple payments? I guess that's a scenario possible to reach by creating payments from /admin

Reply:
you could do
`order.payments.sum(&:amount)`
but that would also be incorrect because it would sum failed payments with finalized payments.
we would need a rule to use payment_total and, if no finalized payments are available, use the uncaptured payment.

could be
`(order.payment_total.nil? || order.payment_total.zero?) ? order.payments.first.amount : order.payment_total `